src/content/docs/infrastructure/infrastructure-ui-pages/hosts-new-view.mdx

```diff
@@ -0,0 +1,173 @@
+
+    <img
+        title="Hosts System page"
+        alt="Hosts System page"
+        src={infrastructureHostNewSystem}
+    />
+
+    <figcaption>
+        **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > Hosts (New) > System**
+    </figcaption>
+
+    </Collapser>
+
+    <Collapser
+    id="network"
+    title="Network"
+    >
+    The **Network** tab gives you the bandwidth and error data about your network interfaces.
+
+    In the **Network** table, you can filter your hosts by metric name. With the <img src={infrastructureExportCSV} class="inline" style={{height: "30px", width: "30px"}} alt="Export CSV" title="Export CSV" /> button, you can export this table as a CSV file.
+
+    <img
+        title="Hosts Network page"
+        alt="Hosts Network page"
+        src={infrastructureHostNewNetwork}
+    />
+
+    <figcaption>
+        **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > Hosts (New) > Network**
+    </figcaption>
+
+    </Collapser>
+
+    <Collapser
+    id="processes"
+    title="Processes"
+    >
+    The **Processes** tab gives you data about CPU percentage, I/O bytes, and memory usage for individual or groups of processes. 
+
+    In the **Processes** table, you can filter your hosts by metric name. With the <img src={infrastructureExportCSV} class="inline" style={{height: "30px", width: "30px"}} alt="Export CSV" title="Export CSV" /> button, you can export this table as a CSV file.
+
+    <img
+        title="Hosts Processes page"
+        alt="Hosts Processes page"
+        src={infrastructureHostNewProcesses}
+    />
+
+    <figcaption>
+        **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > Hosts (New) > Process**
+    </figcaption>
+
+    </Collapser>
+
+    <Collapser
+    id="storage"
+    title="Storage"
+    >
+    The **Storage** tab gives you the resources' capacity and efficiency, including your devices' overall utilization, disk usage, or I/O operations. 
+
+    In the **Storage** table, you can filter your hosts by metric name. With the <img src={infrastructureExportCSV} class="inline" style={{height: "30px", width: "30px"}} alt="Export CSV" title="Export CSV" /> button, you can export this table as a CSV file.
+
+    <img
+        title="Hosts Storage page"
+        alt="Hosts Storage page"
+        src={infrastructureHostNewStorage}
+    />
+
+    <figcaption>
+        **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > Hosts (New) > Storage**
+    </figcaption>
+
+    </Collapser>
+
+    <Collapser
+    id="containers"
+    title="Containers"
+    >
+    The **Containers** tab gives you the CPU percentage, I/O bytes, and memory usage for individual or group of containers. 
```

Comment:
I think you need an "s" at the end of "group": The **Containers** tab gives you the CPU percentage, I/O bytes, and memory usage for individual or groups of containers.